examples/experimental/tongye/opacus_with_Plan/Opacus_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
<p>line 38: this is more pythonic: <code>data, target = data_tuple</code>.  You could also do this in the for loop directly:</p><pre class="ql-syntax" spellcheck="false">for _batch_idx, (data, target) in enumerate(dl):
</pre><p> </p><p>Same for line 48:</p><p> </p><p><code>eps, best_alpha = privacy_engine_ptr.get_privacy_spent(args["delta"])</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>thanks</p>

Reply:
<p>Sorry, this is a hangover from before we had Tuples and destructuring so its not intentional, we need to make sure this still works as expected because destructuring might be a little bit magic..... Actually I just tested and while destructuring does work it also seems to cause some small issue with the GC, so ill go make a ticket for that now. Thanks. 😊</p>